pkgs/top-level/all-packages.nix

```diff
@@ -34491,6 +34493,14 @@ with pkgs;
 
   wordpress = callPackage ../servers/web-apps/wordpress { };
 
+  wordpressPackages = callPackage ../servers/web-apps/wordpress/themes-and-plugins {
+    plugins = lib.importJSON ../servers/web-apps/wordpress/themes-and-plugins/plugins.json;
+    themes = lib.importJSON ../servers/web-apps/wordpress/themes-and-plugins/themes.json;
+    languages = lib.importJSON ../servers/web-apps/wordpress/themes-and-plugins/languages.json;
+    pluginLanguages = lib.importJSON ../servers/web-apps/wordpress/themes-and-plugins/pluginLanguages.json;
+    themeLanguages = lib.importJSON ../servers/web-apps/wordpress/themes-and-plugins/themeLanguages.json;
```

Comment:
language json files are missing in this PR. should we include the empty json files or is this safe when the file is missing?

Reply:
I think I'll leave it. As nix is a lazy language it doesn't matter and it creates some visibility that this feature is still missing. Don't have any hard opinion on this though.